fiftyone/utils/openimages.py

```diff
@@ -596,9 +675,9 @@ def _setup(
     seed=None,
     download=False,
 ):
-    did_download = False
-    _label_types = _parse_label_types(label_types)
 
+    did_download = False
+    _label_types = label_types
```

Comment:
This should use `_parse_label_types()` and have the version passed into `_setup(..., version, ...)`.